rbi/core/object.rbi

```diff
@@ -27,36 +27,4 @@
 class Object < BasicObject
   include Kernel
 
-  # Returns an integer identifier for `obj`.
```

Comment:
I wasn't sure if I should delete this file or its still useful as is 🤔

Reply:
I think this is the only file that declares to Sorbet that `class Object` has `include Kernel`, so I think it's important to keep this.